lib/po.js

```diff
@@ -244,6 +263,7 @@ PO.Item = function () {
     this.extractedComments = [];
     this.flags = {};
     this.obsolete = false;
+    this.nplurals = Number(nplurals);
```

Comment:
Good call. No reason for the fallback to 2 to happen inside `toString()` every time.